### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by ddzuba**

### CODE REVIEW

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

File: app/models/util/DsaToAkkaCoder.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+  def processParts(path: String, separator: String, proc:(String) => String) =
+    path.split(separator).map(proc(_)).mkString(separator)
+
+  implicit class DsaToAkkaCoderFuncs(val s: String) {
+    def forAkka = dsaToAkkaPath(s)
+    def forDsa = akkaToDsaPath(s)
+  }
+}
```
Comment (dmitriyisaev) [inline app/models/util/DsaToAkkaCoder.scala:30]: This should be covered with unit tests

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Review by uralian — APPROVED** [review]

**Pull request merged**